PROFILER.md

```diff
@@ -9,21 +9,25 @@ which inspects the state of all threads to determine how long they have been
 running for. This does not require any code to be modified to support profiling
 so that any code which runs will be profiled automatically.
 
-The profiler can be enabled by setting the following:
+## Enabling Profiling
 
- * `com.iopipe.plugin.profiler` to `true`
- * `IOPIPE_PROFILER_ENABLE` to `true`
+Set the system property `com.iopipe.plugin.profiler` to `true`
 
-The profiler may be customed by setting the specified properties and/or
+### For Serverless Framework:
+
+In the `serverless.yml` file, under `functions:`, `(your_function_name_here):`, `environment:`, set the environmental variable `IOPIPE_PROFILER_ENABLE` to `true`
+
+## Customizing Profiling
+
+The profiler may be customized by setting the specified properties and/or
 environment variables:
 
  * The duration of time between each sample, in microseconds:
    * The default is 1,000 microseconds (1ms).
-   * `com.iopipe.plugin.profiler.samplerate`
-   * `IOPIPE_PROFILER_SAMPLERATE`
+   * `IOPIPE_PROFILER_SAMPLERATE` in the environmental varibles of the `serverless.yml` file, or the environment variable `com.iopipe.plugin.profiler.samplerate`.
```

Comment:
varibles => variables and `com.iopipe....` is not an environment variable, you can drop the mention if you don't want to cover system properties.